Dockerfile

```diff
@@ -7,12 +7,20 @@ WORKDIR /srv/node
 
 RUN buildDeps=' \
     git \
+    yarn \
     ' && \
+    # `apt-transport-https` is required to use https deb repositories
+    apt-get update -y && \
```

Comment:
This already happens down below. Did you mean to delete the other one?

Reply:
that was not a mistake. I need to fetch `apt-transport-https` and install it first. otherwise I cannot do anything because the `https://` dep repo will not be supported. So, I need to install it separately, hence these redundant commands.